core/templates/components/skiplinks/skiplinks.twig

```diff
@@ -0,0 +1,9 @@
+{% set skipID = "skip-to-content-" ~ random() %}
+
+<header>
+  <a href="#{{ target_id|default(skipID) }}" class="su-skiplinks">{{ link_text|default('Skip to main content') }}</a>
+</header>
+
+<main>
```

Comment:
Do we need to have the target in this template file? It won't be directly usable if this main markup is here. I am assuming this is an example. I would like to see a directly usable option as well as the demo.

Reply:
@sherakama is your recommendation that the twig template only include the following?

```
{% set skipID = "skip-to-content-" ~ random() %}

<a{{ attributes }} href="#{{ target_id|default(skipID) }}" class="su-skiplinks {{ modifier_class }}">{{ link_text|default('Skip to main content') }}</a>
```

And, then, perhaps add exampel markup directly to the SCSS file, something like:

```
<a href="#main-content" class="su-skiplinks ">Skip to main content</a><main><h2 id="main-content" class="su-sr-only-text" tabindex="-1">Main Content</h2></main>
```

Reply:
I think skiplinks are a special snowflake. They are, by definition, not self-contained. I don't believe there's anyway to get them to work with a single twig file.

Since they'll require the user to create custom markup anyway, I vote for having all the necessary pieces in our `.twig` file, and document how users need to use each piece. I would therefore vote for restoring the `<header>` wrapper around the link itself, since it demonstrates the recommended placement for the link.